partials/new_device_login.html

```diff
@@ -1,4 +1,8 @@
 <% extends "partials/base/base_text_only.html" %>
 
 <% block heading %>{{t "New sign-in to Firefox"}}<% endblock %>
-<% block text_primary %>{{{ device }}}<br/>{{ timestamp }}<% endblock %>
+<% block text_primary %>{{{ device }}}<br/>
+<span style="display:block;">IP address: {{ ip }}</span>
```

Comment:
I'm not sure how the [t helper function](https://github.com/mozilla/fxa-auth-mailer/blob/master/templates/index.js#L6) goes about its thing. You might have to [dig into the handlebars documentation](http://handlebarsjs.com/) to figure out whether t does what we need it to, and if not, how to make it do so.